Formula/netpbm.rb

```diff
@@ -22,6 +22,7 @@ class Netpbm < Formula
   depends_on "libtiff"
   unless OS.mac?
     depends_on "flex"
```

Comment:
Should this dep be `:build`?
```suggestion
    depends_on "flex" => :build
```